mycore-mets/src/main/java/org/mycore/mets/model/MCRMETSGeneratorFactory.java

```diff
@@ -44,21 +44,21 @@
  */
 public final class MCRMETSGeneratorFactory {
 
-    private static MCRMETSGeneratorSelector GENERATOR_SELECTOR;
+    private static MCRMETSGeneratorSelector generatorSelector;
 
-    private static boolean IGNORE_METS_XML;
+    private static boolean ignoreMetsXml;
```

Comment:
should be final resp. upper case

Reply:
Can't be final because it has multiple assignments. One in try, one in the catch case.